cluster-scope/base/external-secrets.io/externalsecrets/lokistack-config-custom/externalsecret.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+namespace: openshift-operators-redhat
```

Comment:
`namespace` should only be under `metadata`. Specifying it here makes this invalid and this manifest won't be deployed.